lib/boku/client.py

```diff
@@ -328,10 +339,25 @@ def _get(self, url):
         with statsd.timer('solitude.boku.api'):
             return requests.get(proxy)
 
+    def check_sig(self, data):
+        """
+        Check that a signature is valid. This check has to be done on the
+        proxy, because the solitude database does not have access to the
+        required BOKU_SECRET_KEY.
+        """
+        try:
+            self.api_call('/check_sig', data, add_timestamp=False)
```

Comment:
It's not a Fake API call, its a call to the Proxy. The ProxyClient is run by the database instance to interact with the Boku proxy.